spec/components/provider_interface/completed_offer_summary_component_spec.rb

```diff
@@ -115,6 +117,8 @@ def row_link_selector(row_number)
     expect(row_text_selector(:location, render)).to include(course_option.site.name_and_address)
     expect(row_text_selector(:full_or_part_time, render)).to include(course_option.study_mode.humanize)
     expect(row_text_selector(:accredited_provider, render)).to include(course_option.course.accredited_provider.name_and_code)
+    expect(row_text_selector(:qualification, render)).to include(course_option.course.qualifications.reverse.map(&:upcase).join(' with '))
```

Comment:
can we be explicit in the test, instead of copying the implementation, as in 'PGCE with QTS', same below ?